bindings/python/crocoddyl/core/core.cpp

```diff
@@ -1,7 +1,8 @@
 ///////////////////////////////////////////////////////////////////////////////
 // BSD 3-Clause License
 //
-// Copyright (C) 2018-2020, University of Edinburgh, LAAS-CNRS
+// Copyright (C) 2018-2021, University of Edinburgh
```

Comment:
The copyright is not correct for Edinburgh. 2018 is not correct.

Reply:
I have discussed with @nim65s about the copyright information.
This file, as each one defined in the c++ library, needs to start from 2019 (including LAAS-CNRS).

As @nim65s explained me, these files were not a copy-pasted from 2018 library as this library was purely written in Python.
It is the same for the code that defines the algorithms (e.g. SolverDDP) as they were not copy-pasted neither (I did rewritten them).

Remember that I worked in 2019 for LAAS and Edinburgh.
So, there are many files where both institutions share the copy-holders from 2019.

This requires to modify the copyright information for the entire library, not only the files in this PR.
Thus, I will create another PR only focuses on this topic.
Then I will rebase this PR and check again the information for the modified files.

Reply:
I forgot to say that the c++ library was written in 2019 (summertime).
And in autumn, I moved to Edinburgh where I kept working quite a lot on the library.

Reply:
done!